### ISSUE TITLE
Fix parsing errors on webbook homepage and reading interface

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks-book#876]
We have a few parsing errors in our webbook homepage and reading interface.

- [x] duplicate use of id="twitter-logo" on webbook homepage
- [x] div class="tog__toggle" is missing a role attribute
- [x] end tag 'svg' does not match name of current open element 
- [x] Not closing our `<section>` element on reading interface. Header is being opened before section is closed.

- [x] Validation errors related to the contributors blade template (labelled-by doesn't reference the right ID and multiple contributors use the same SVG id)
